### DISCUSSION TITLE
Layer folders

### DISCUSSION THREAD

**Discussion by Nukiloco** [opentoonz/opentoonz#6822] [Ideas]
**Is your feature request related to a problem? Please describe**
There doesn't seem to be any way to organize a bunch of layers into a group/folder in the xsheet/timeline unless if a separate scene is made.

**Describe the solution you'd like**
I would want it possible to be able to make a folder layer in the xsheet/timeline where it allows me to group up layers into that folder. I would also like the folder layer to be renamable and when you set the opacity or filter, it applies to all of the other layers underneath without changing their opacity or filter of the layer.

**Comment by konero** [comment]
@Flairieve Select the columns you want to group and right-click > Collapse.

**Comment by age3rcm** [comment]
actually, this does not resolve the problem of organization konero
collapsing the columns create a sub-xsheet and the columns will be "locked" inside
whereas the "folder" is just to organize the columns while also keeping them free to manipulate
one important thing is also the z space, you can make different columns be on top and others below it in a way that you can make it on separate columns instead of the 5 locked layers in a sub x-sheet

this will improve the capability of opentoonz in regards to cut-out animation

**Comment by gab3d** [comment]
The concept of layer folders is a known one, and for sure it has its advantages for some organizational tasks.
Currently there is not such feature in OT, but certainly nobody is against its existence.
If/when there is a developer willing to devote time and efforts to making it a reality, for sure we will all be applauding him/her. :smile:

**Comment by age3rcm** [comment]
gab3d
opentoonz is abandoned by coders?
people are dropping it and forgetting about it?
i am worried

**Comment by artisteacher** [comment]
@shun-iwasawa is still going strong. There have been a lot of fixes lately. While there are a lot of new features I would love to see implemented, there are some stability issues that really do need to take priority.

Though personally as a teacher, I find Tahoma2D a lot easier for my students. Especially since they need to be setup to work remotely some days.

**Comment by gab3d** [comment]
@age3rcm 
No, OpenToonz is alive.
There are some characteristics of its development that makes some people think there is no or little movement behind it, though.
There are not much strong developers behind it, that's a fact; it has an annual release cycle, which makes some people think the software is kind of abandoned, and finally there is somewhat of a lack of community growing directives/actions by Dwango/Ghibli, whom are mainly concerned by its own use of the software, rather than making its use widen among general people.
I have not much to criticize though, but that's how things are...

Tahoma2D is an effort by a very valuable developer to gain independence in the development of some features that would take longer time under the umbrella of the official OpenToonz, as well as the Morevna Project has its own version of the software and development too.

**Comment by RodneyBaker** [comment]
While I like the idea of 'Folders' within the xsheet there are some competing equivalencies that we should take into account.
For instance, in the Schematic we can Group layers/columns/nodes as well as collapse them into sub-xsheets.
Is a 'Layer Folder' the same thing as a column layer grouping?
I will guess it is although a Folder might be a more arbitrary (macro) container used expressly for the purpose of organizing groups of objects.

At any rate, while Layer Folders will be useful it is important to consider this in the larger context of other UI management schemes users will encounter.

Aside:  I believe there is another request open for similar column folder functionality but a quick search did not reveal it.

**Comment by age3rcm** [comment]
*Is a 'Layer Folder' the same thing as a column layer grouping?
yes, it would work as the same as in the schematic, but if possible, to be able to copy paste cells and keyframes of the entire folder to another frame of the same grouped column
